martin/src/pg/query_tables.rs

```diff
@@ -108,25 +109,37 @@ pub async fn table_to_query(
     bounds_type: BoundsCalcType,
     max_feature_count: Option<usize>,
 ) -> PgResult<(String, PgSqlInfo, TableInfo)> {
-    let schema = escape_identifier(&info.schema);
-    let table = escape_identifier(&info.table);
-    let geometry_column = escape_identifier(&info.geometry_column);
     let srid = info.srid;
 
     if info.bounds.is_none() {
         match bounds_type {
             BoundsCalcType::Skip => {}
             BoundsCalcType::Calc => {
                 debug!("Computing {} table bounds for {id}", info.format_id());
-                info.bounds = calc_bounds(&pool, &schema, &table, &geometry_column, srid).await?;
+                info.bounds = calc_bounds(
+                    &pool,
+                    &info.schema,
+                    &info.table,
+                    &info.geometry_column,
```

Comment:
Might as well just pass a ref to `info`

```suggestion
                    &info
```

Reply:
Well..
These are not identical and the reason for it being this way is a few lines further down:

![image](https://github.com/user-attachments/assets/ff828f01-f431-4563-8f85-cd750c6dbab5)
![image](https://github.com/user-attachments/assets/dc54500c-569a-48e0-90f9-0292ed267075)

=> since bounds was not borrowed above, this worked before the suggested change ^^

Reply:
=> reverted in https://github.com/maplibre/martin/pull/1220/commits/daae7f2023a76735205a2ae0c4f2544e9afdc01d

Reply:
@CommanderStorm see https://github.com/sharkAndshark/martin/pull/149